client/html/themes/elegance/aimeos.js

```diff
@@ -4,3 +4,56 @@
  * @license LGPLv3, http://opensource.org/licenses/LGPL-3.0
  * @copyright Aimeos (aimeos.org), 2014
  */
+
+
+
+/**
+ * Hides or shows the catalog filter search reset button
+ * depending on if the input text field is empty or not.
+ */
+AimeosCatalogFilter.hideOrShowCatalogFilterTextSearchResetButton = function() {
+	if ($(this).val() !== '')
+		$(this.nextElementSibling).css('display', 'inline-block');
```

Comment:
My opinion. Usually people hate me when I add them. So in this project I can add them. Great.